LLaMA_LoRA.ipynb

```diff
@@ -60,20 +60,15 @@
         "# @title A small workaround { display-mode: \"form\" }\n",
         "# @markdown Don't panic if you see an error here. Just click the `RESTART RUNTIME` button in the output below, then Run All again.\n",
         "# @markdown The error will disappear on the next run.\n",
-        "!pip install Pillow==9.3.0 numpy==1.23.5\n",
+        "%pip install Pillow==9.3.0 numpy==1.23.5\n",
         "\n",
+        "import pkg_resources as r\n",
         "import PIL\n",
-        "major, minor = map(float, PIL.__version__.split(\".\")[:2])\n",
-        "version_float = major + minor / 10**len(str(minor))\n",
-        "print('PIL', version_float)\n",
-        "if version_float < 9.003:\n",
-        "    raise Exception(\"Restart the runtime by clicking the 'RESTART RUNTIME' button above (or Runtime > Restart Runtime).\")\n",
-        "\n",
         "import numpy\n",
-        "major, minor = map(float, numpy.__version__.split(\".\")[:2])\n",
-        "version_float = major + minor / 10**len(str(minor))\n",
-        "print('numpy', version_float)\n",
-        "if version_float < 1.0023:\n",
+        "for module, min_version in [(PIL, \"9.3\"), (numpy, \"1.23\")]:\n",
```

Comment:
repeat the versions, and iterate to check if they're loaded

Reply:
Nice!